speechmatics_flow/cli.py

```diff
@@ -99,18 +99,23 @@ def get_conversation_config(
     :param args: Keyword arguments probably from the command line.
     :type args: Dict
 
-    :return: Settings for the ASR engine.
+    :return: Settings for the Flow engine.
     :rtype: models.ConversationConfig
     """
 
-    config: Dict[str, Any] = {}
+    config = {"template_id": "default"}
```

Comment:
```suggestion
    config = {}
```
It will be overridden if the user uses a config file and the default will then come from `ConversationConfig` model. Let's not have the default defined in multiple places.